.ci/e2e-expected/backslash-dt.txt

```diff
@@ -4,7 +4,10 @@
  testdb_e2e_psql_v?? | public             | users                           | BASE TABLE |                              |                      |                           |                          |                        |                    |          |               |                   |                  | COMMITTED     |                 |
  testdb_e2e_psql_v?? | pg_catalog         | pg_class                        | VIEW       |                              |                      |                           |                          |                        |                    |          |               |                   |                  |               |                 |
  testdb_e2e_psql_v?? | pg_catalog         | pg_index                        | VIEW       |                              |                      |                           |                          |                        |                    |          |               |                   |                  |               |                 |
+ testdb_e2e_psql_v11 | pg_catalog         | pg_indexes                      | VIEW       |                              |                      |                           |                          |                        |                    |          |               |                   |                  |               |                 | 
```

Comment:
For consistency, should we change it to "testdb_e2e_psql_v??" ?
Perhaps update `sed -i "s/testdb_e2e_psql_v../testdb_e2e_psql_v$PSQL_VERSION/" .ci/e2e-expected/backslash-dt.txt` to capture only question mark rather than any character(represented by period).